wamp-stack.sls

```diff
@@ -1,19 +1,14 @@
+{% set SOURCE_PATH = 'https://downloads.bitnami.com/files/stacks/wampstack/' %}
+
 wamp-stack:
-  '5.5.30.0':
-    full_name: 'Bitnami WAMP Stack 5.5.30'
-    installer: 'https://downloads.bitnami.com/files/stacks/wampstack/5.5.30-0/bitnami-wampstack-5.5.30-0-windows-installer.exe'
-    install_flags: '--mode unattended --base_password salt-installer-secret --apache_server_port 8080'
-    uninstaller: 'c:\Bitnami\wampstack-5.5.30-0\uninstall.exe'
-    uninstall_flags: '--mode unattended'
-    msiexec: False
-    locale: en_US
-    reboot: False
-  '5.4.36.0':
-    full_name: 'Bitnami WAMP Stack 5.4.36'
-    installer: 'https://downloads.bitnami.com/files/stacks/wampstack/5.4.36-0/bitnami-wampstack-5.4.36-0-windows-installer.exe'
-    install_flags: '--mode unattended --base_password salt-installer-secret --apache_server_port 8080'
-    uninstaller: 'c:\Bitnami\wampstack-5.4.36-0\uninstall.exe'
+  {% for VERSION in ['7.1.27-1', '5.6.40-1'] %}
```

Comment:
Are the older versions no longer available? If so, please add those too.

Reply:
according to the website only officially these two stable versions
https://bitnami.com/stack/wamp/installer

Reply:
Sounds good